src/main/java/touro/chess/ChessController.java

```diff
@@ -0,0 +1,61 @@
+package touro.chess;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.image.ImageView;
+
+import java.util.ArrayList;
+
+public class ChessController {
+
+    @FXML
+    ArrayList<Label> squares;
+
+    public void initialize() {
+        setupInitialBoard();
+//        updateImage('b', 0, 30, Piece.rook);
+    }
+
+    private void setupInitialBoard() {
+        squares.get(0).setStyle("-fx-background-image: url(\"/images/b_rook.png\");");
```

Comment:
In general, you shouldn't put CSS in java code. Escape characters make things difficult to read.

Instead, create a CSS class for each piece (and one for "empty") and add the styleClass of the Label with it.

Reply:
All fixed in new commit